benches/storage/scan.rs

```diff
@@ -61,31 +55,171 @@ fn bench_tombstone_scan(b: &mut Bencher) {
                 k.clone(),
                 ts,
             )
-            .expect("");
+            .unwrap();
         store
             .commit(
                 Context::new(),
                 vec![Key::from_raw(&k)],
                 ts,
                 ts_generator.next().unwrap(),
             )
-            .expect("");
+            .unwrap();
     }
 
+    // We should got nothing when scanning any keys.
+
     kvs = KvGenerator::new(100, 1000);
+    let ts = ts_generator.next().unwrap();
     b.iter(|| {
         let (k, _) = kvs.next().unwrap();
         assert!(
             store
                 .scan(
-                    Context::new(),
+                    super::util::new_no_cache_context(),
                     Key::from_raw(&k),
                     1,
                     false,
-                    ts_generator.next().unwrap()
+                    ts
                 )
                 .unwrap()
                 .is_empty()
         )
     })
 }
+
+fn bench_scan(b: &mut Bencher, forward_scan: bool, versions: usize, long_value: bool) {
+    const NUMBER_KEYS: usize = 10000;
+
+    let path = TempDir::new("").unwrap();
+    let store = super::util::new_storage(path.path().to_str().unwrap());
+
+    println!("Preparing data...");
+
+    let value_length = if long_value {
```

Comment:
I think you'd better make value_length more representative, for example 32/100/200bytes.